torch/_inductor/codegen/cpp.py

```diff
@@ -770,7 +770,13 @@ def floordiv(a, b):
         # a and b are integer type
         quot = f"{a} / {b}"
         rem = f"{a} % {b}"
-        return f"(({a} < 0) != ({b} < 0) ? ({rem} != 0 ? {quot} - 1 : {quot}) : {quot})"
+        code = BracesBuffer()
+        code.writeline("[&]()")
+        with code.indent():
+            code.writeline(f'AOTI_TORCH_CHECK({b}!=0, "ZeroDivisionError");')
```

Comment:
Feels we shouldn't hardcode to use `AOTI_TORCH_CHECK`

Reply:
I agree, but the `self.assert_function` is defined in `CppVecKernel`, which cannot be directly used in static method of `CppOverrides`. I am not familiar with how PyTorch team handles these macros, What's your advice on handling this issue?

Reply:
can we use `V.kernel.assert_function`?